doc/data/messages/t/too-many-locals/good.py

```diff
@@ -1 +1,30 @@
-# This is a placeholder for correct code for this message.
+class TreeNode(object):
```

Comment:
I don't understand this example there's a lot of  noise unrelated with the pylint message.

Reply:
This example was from a programming exercise from w3schools about binary search trees.
https://www.w3resource.com/python-exercises/data-structures-and-algorithms/python-binary-search-tree-index.php